companion/src/CMakeLists.txt

```diff
@@ -300,12 +300,19 @@ endif()
 
 ############# Install ####################
 
+# Generate list of simulator plugins, used by all platforms
+file(GLOB simulator_plugins "${CMAKE_BINARY_DIR}/*opentx-*simulator${CMAKE_SHARED_LIBRARY_SUFFIX}")
+# the current flavour is not automatically added if build in the current cmake iteration, so always
+# add its library name to be sure
+string(TOLOWER ${PCB} FLAVOUR)
+get_property(current_plugin TARGET opentx-${FLAVOUR}-simulator PROPERTY LOCATION)
+list(APPEND simulator_plugins "${current_plugin}")
```

Comment:
running cmake a second time is even more of a hack. 

I first wrapped the file global into a INSTALL(CODE fragment but that looked even more horrible and forced more stuff inside the INSTALL(CODE fragment